src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -105,8 +113,8 @@ public static TokenValidationParameters CreateTokenValidationParameters()
             if (signingKeys.Length > 0)
             {
                 result.IssuerSigningKeys = signingKeys;
-                result.ValidateAudience = true;
```

Comment:
These no longer need to be set to true. When explicit validators are specified, these settings are ignored.